code/game/objects/items/RCD.dm

```diff
@@ -710,7 +711,10 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 				icon = design["icon"]
 			icon = replacetext(icon," ","-") //css class names cannot have spaces in them
 
-			designs += list(list("design_id" = i, "selected" = ((design["title"] == design_title) && (sub_category == design_category)), "title" = design["title"], "icon" = icon))
+			transform = "scale(1.0)"
+			if(icon == "window0" || icon == "rwindow0")
+				transform = "scale(0.7)"
```

Comment:
Transform is better to be a function in js to reduce the amount of data that is sent to the client.

Reply:
I believe `selected` can also be computed client-side, also to reduce the amount of data sent.

Reply:
Ill make it a constant expression in the tsx file for this check

Reply:
Done